README.md

```diff
@@ -26,6 +26,15 @@ timdex_dataset_api = {git = "https://github.com/MITLibraries/timdex-dataset-api.
 ... other dependencies...
 ```
 
+## Environment Variables
+
+### Required
+
+### Optional
+```shell
+TDA_LOG_LEVEL=# log level for timdex-dataset-api, accepts [DEBUG, INFO, WARNING, ERROR], default INFO
```

Comment:
For clarification, is the purpose of this env var so that:

if the calling application is configured to log level INFO, messages logged by `timdex-dataset-api` functions at the DEBUG level still show up as long as `TDA_LOG_LEVEL=DEBUG`? 🤔

Reply:
That's correct!  And confirmed locally when installing as a library for `Transmogrifier`.  I had to explicitly set `TDA_LOG_LEVEL=DEBUG` to get logging.  Whether that's ideal or not... TBD, but that's how it works.